server/server.go

```diff
@@ -409,6 +408,8 @@ func (s *Server) ListStores(ctx context.Context, req *openfgapb.ListStoresReques
 	ctx, span := s.tracer.Start(ctx, "listStores")
 	defer span.End()
 
+	time.Sleep(2 * time.Second)
```

Comment:
This crept in?

Reply:
Oops. lol

I was testing out the request timeout. Good catch :)